torchvision/csrc/ops/quantized/cpu/qroi_align_kernel.cpp

```diff
@@ -8,6 +8,9 @@ namespace ops {
 
 namespace {
 
+// FIXME: android is no longer supported and thus the comments below can be
+// acted on
+
```

Comment:
Not sure we want to remove that comment - it gives context on why the FIXME hasn't been done yet

Reply:
I've removed this again since we aren't purging mobile anyway now.